src/components/pop-up/index.tsx

```diff
@@ -0,0 +1,57 @@
+import styled from '@emotion/styled'
```

Comment:
I think we are naming components in PascalCase and not kebab-case 🤔

Reply:
yes but not for the folders thu. the component name is in PascalCase but the folder is in Kebab-case (I was following the website's engineering thu. not my idea. ) ... I agree with you on this

Reply:
This can be solved at a later date then, disregard this comment

Reply:
> yes but not for the folders thu. the component name is in PascalCase but the folder is in Kebab-case (I was following the website's engineering thu. not my idea. ) ... I agree with you on this

The kebab case Directories was a mistake by Braydon (@nobrayner),  folders will preferably be PascalCase

Reply:
They can arguably be all lowercase folders however the files are 100% PascalCase
https://reactjs.org/docs/faq-structure.html

Reply:
same